### ISSUE TITLE
FR add patterns for substituting Affine into Gaussian

### ISSUE THREAD

**Issue by fritzo** [pyro-ppl/funsor#237]
Blocked by #157 (so as not to proliferate deprecated `Affine` patterns)

This came up in @jpchen's biased kalman filter experiment, where we are using `matrix_and_mvn_to_funsor()` to create a `Joint`, then substituting `data + bias` into this distribution. Currently `Affine` patterns exist for `dist.Normal` funsors, but not raw `Gaussian`s.

**Comment by fritzo** [comment]
Closing as duplicate of #72
